app/renderer/js/utils/enterprise-util.ts

```diff
@@ -0,0 +1,69 @@
+import fs = require('fs');
+import path = require('path');
+
+import Logger = require('./logger-util');
+
+const logger = new Logger({
+	file: 'enterpise-util.log',
+	timestamp: true
+});
+
+let instance: null | EnterpriseUtil = null;
+
+class EnterpriseUtil {
+	enterpriseSettings: any;
```

Comment:
Once the settings we'll allow are finalized, we'll need to remove this any.

Reply:
Yeah, I'll leave a todo comment here for future work.